extensions/jitsi/common/rnnoise/COPYRIGHT.md

```diff
@@ -0,0 +1,3 @@
+`RnnoiseProcessor.js` and `index.js` are reproduced from the [official Jitsi Meet client](https://github.com/jitsi/jitsi-meet/blob/master/react/features/stream-effects/rnnoise), published under [Apache License 2.0](https://github.com/jitsi/jitsi-meet/blob/master/LICENSE).
```

Comment:
I would recommend using submodule and symbolic links to add these files so that we know what revisions they're from.